interface/forms/group_attendance/new.php

```diff
@@ -0,0 +1,254 @@
+    <script src="<?php echo $GLOBALS['assets_static_relative'];?>/jquery-min-1-9-1/index.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'];?>/jquery-ui-1-10-4/ui/jquery.ui.core.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'];?>/datatables.net-1-10-11/js/jquery.dataTables.min.js"></script>
+    <script src="<?php echo $GLOBALS['web_root'];?>/library/dialog.js"></script>
+</head>
+
+<body class="body_top">
+<?php if($form_id){ ?>
+<form id="group_attendance_form" method=post onclick="top.restoreSession();" action="<?php echo $rootdir;?>/forms/group_attendance/save.php?mode=update&id=<?php echo attr($form_id) ;?>" name="my_form">
+<?php } else { ?>
+<form id="group_attendance_form" method=post onclick="top.restoreSession();" action="<?php echo $rootdir;?>/forms/group_attendance/save.php?mode=new" name="my_form">
+<?php } ?>
+    <div id="add_participant">
+        <div class="button_wrap">
+            <input class="button-css add_button" type="button" value="<?php echo xla('Add'); ?>">
+        </div>
+        <div id="add_participant_element"  style="display: none;">
+            <div class="patient_wrap">
+                <span class="input_label"><?php echo xlt("Patient Name");?></span>
+                <input name="new_id" class="button-css new_patient_id" type="hidden" value="">
+                <input name="new_patient" class="button-css new_patient" type="text" value=""  readonly>
+                <div class="error_wrap">
+                    <span class="error"></span>
+                </div>
+            </div>
+            <div class="comment_wrap">
+                <span class="input_label"><?php echo xlt("Comment");?></span>
+                <input name="new_comment" class="button-css new_comment" type="text" value="">
+            </div>
+            <div class="button_wrap">
+                <input class="button-css add_patient_button" type="button" value="<?php echo xla('Add Patient'); ?>">
+                <input class="button-css cancel_button" type="button" value="<?php echo xla('Cancel'); ?>" >
+            </div>
+        </div>
+    </div>
+    <table id="group_attendance_form_table">
+        <thead>
+        <tr>
+            <th align="center"><?php echo xlt('Participant’s name'); ?></th>
+            <th align="center"><?php echo xlt('Patient’s number'); ?></th>
+            <th align="center"><?php echo xlt('Status in the meeting'); ?></th>
+            <th align="center"><?php echo xlt('Comment'); ?></th>
+        </tr>
+        </thead>
+        <tbody>
+        <?php foreach ($participants as $participant){?>
+            <tr>
+                <td ><?php echo text($participant['fname'] . ", " . $participant['lname']); ?></td>
+                <td ><?php echo text($participant['pid']); ?></td>
+                <td >
+                    <select class="status_select" name="<?php echo "patientData[" . $participant['pid'] . "][status]" ;?>">
```

Comment:
doesn't $participant['pid'] need to be in php tags an echoed?
if so, also needs to be wrapped with attr()